components/profile/components/EditProfile/index.js

```diff
@@ -27,27 +27,68 @@ const {
 const { plus } = ICONS;
 
 class EditProfile extends Component {
-    state = { 
-        fileList: [], 
-        location: null, 
-        locationLoading: false, 
-        previewImage: '', 
-        previewVisible: false, 
+    constructor(props){
+        super(props);
+
+        const { firstName, lastName, bio, highSchool, intendedUni, username } = props.user;
```

Comment:
no need to set these values on the state, you already have access to it on the `props`

Reply:
Also if you wanted . to do default syntax it is as follows:
```
const { firstName = '', lastName = '', bio = '' ... } = props.user;
```

Reply:
True, but it's the Edit Profile page, I am populating the respective fields with the existing info, then users can decide to change the values. Props can't be edited, so I use state instead.
Basically, my flow is =>
-  control form based on values in state
- populate state initially from props

Reply:
dont store this on the component then, store it in the redux so that if you need that information anywhere else you can access it

Reply:
I'm already storing the user information in redux store. This set of data is only used in the EditProfile page, so I don't think there will be a need to put it in redux store 🤔